docs/book/src/developer/tilt.md

```diff
@@ -55,6 +55,21 @@ Add following extra_args to log Power VS REST API Requests/Responses
    }
 }
 ```
+
+To deploy workload cluster with [Power VS cloud controller manager](/topics/powervs/external-cloud-provider.html) which is currently in experimental stage, Set `POWERVS_PROVIDER_ID_FORMAT` to `v2` under kustomize_substitutions.
+
+```json
+{
+   "default_registry": "gcr.io/you-project-name-here",
+   "provider_repos": ["../cluster-api-provider-ibmcloud"],
+   "enable_providers": ["ibmcloud", "kubeadm-bootstrap", "kubeadm-control-plane"],
+   "kustomize_substitutions": {
+      "IBMCLOUD_API_KEY": "XXXXXXXXXXXXXXXXXX",
+      "POWERVS_PROVIDER_ID_FORMAT": "v2"
```

Comment:
are we going to add anywhere else for this option to override this option with clusterctl flow?

Reply:
With clusterctl flow, as like we export `IBMCLOUD_API_KEY`, we need to export `POWERVS_PROVIDER_ID_FORMAT`
```

export POWERVS_PROVIDER_ID_FORMAT=v2 
export IBMCLOUD_API_KEY=xxxxxxxxxxxxxxx
clusterctl init -i ibmcloud
```

Reply:
true that but that needs to be added in the docs right?

Reply:
Yeah, But in our doc currently where should i add?

Reply:
may be here - https://cluster-api-ibmcloud.sigs.k8s.io/getting-started.html in step 3